_lib/wordpress_view_processor.py

```diff
@@ -51,8 +51,9 @@ def process_view(post):
         hero_url = os.path.expandvars("$WORDPRESS/hero/" + hero_id + "/?json=1")
         response = requests.get(hero_url)
         hero_data = json.loads(response.content)
-        hero_data = hero_data['post']
-        hero_data['related_posts'] = hero_data['custom_fields']['related_post']
-        post['hero'] = hero_data
+        if hero_data['status'] is "ok":
```

Comment:
Line 51 uses double quotes as well. In the JSON output, the status is in double quotes which is why I used them. If you feel that we should still change it to single quotes, I'll do so.